src/utilities/useUSAState.js

```diff
@@ -60,10 +60,12 @@ const useStateCode = ({ useGeoCode }) => {
 export default (options = {}) => {
   const stateCode = useStateCode(options)
   if (!stateCode) return {}
-
   const lowerCaseStateCode = stateCode.toLowerCase()
   const stateName = statesObj[stateCode]
-  const facebookPage = `PSP${stateName.split(' ').join('')}`
+  let facebookPage = `PSP${stateName.split(' ').join('')}`
+  if (stateCode === 'AL') {
+    facebookPage = `${stateName.split(' ').join('')}PSP`
+  }
```

Comment:
This is great!  For anything more complicated, we would probably want to use a state to facebook name mapping object like this:
```
{
  'AL' => 'alabamaPSP`,
  'AZ' => 'PSParazona',
  ...
}
```
But this works great for something this simple.